internal/pods/schedule.go

```diff
@@ -148,12 +148,21 @@ func (np *Scheduled) schedule() error {
 		},
 	}
 
+	// Some containers which run os like rhel/fedora runs tcpdump
+	// as specific user id "72". So it needs pods to be privileged.
+	// Also setting the runAsUser prevent the pods from starting with
+	// random user id
+	privileged := true
+	runAsUser := int64(0)
+
```

Comment:
Can we move this to Line159 (i.e., lets configure the privileges only when the pod is requesting a HostNetwork). 
We use such pods for two use-cases.
1. For identifying the hostname when it does not match with the name in the endpoint
2. As part of diagnose firewall validation 

In the other cases, when this API is called with podNetworking, we use commands like "curl" or "nc" which AFAIU do not require the privilege permissions. What do you say?